src/ng/http.js

```diff
@@ -1420,20 +1420,21 @@ function $HttpProvider() {
       return url;
     }
 
-    function sanitizeJsonpCallbackParam(url, key) {
-      if (/[&?][^=]+=JSON_CALLBACK/.test(url)) {
-        // Throw if the url already contains a reference to JSON_CALLBACK
-        throw $httpMinErr('badjsonp', 'Illegal use of JSON_CALLBACK in url, "{0}"', url);
-      }
-
-      var callbackParamRegex = new RegExp('[&?]' + key + '=');
-      if (callbackParamRegex.test(url)) {
-        // Throw if the callback param was already provided
-        throw $httpMinErr('badjsonp', 'Illegal use of callback param, "{0}", in url, "{1}"', key, url);
-      }
+    function sanitizeJsonpCallbackParam(url, cbKey) {
+      var params = parseKeyValue(url.split('?', 2)[1]);
```

Comment:
I think this will allow things like `...?foo=bar?&JSON_CALLBACK=...`.

Reply:
Hmm, so I guess we should error if there is more than one `?` in the URL?

Reply:
Works for me :+1: